imgur.sh

```diff
@@ -45,17 +45,24 @@ while [ $# -gt 0 ]; do
 	file="$1"
 	shift
 
-	# Check file exists
-	if [ "$file" != "-" -a ! -f "$file" ]; then
-		echo "File '$file' doesn't exist; skipping" >&2
-		errors=true
-		continue
-	fi
-
 	# Upload the image
-	response=$(curl -s -H "Authorization: Client-ID $client_id" -H "Expect: " -F "image=@$file" https://api.imgur.com/3/image.xml) 2>/dev/null
-	# The "Expect: " header is to get around a problem when using this through
-	# the Squid proxy. Not sure if it's a Squid bug or what.
+	# URL -> imgur
```

Comment:
You've put this comment outside the conditional, but the next one (file -> imgur) inside. Be consistent, please.